### DISCUSSION TITLE
Related to GUI

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3193] [Q&A]
Hi,

Is it possbile to open the Nocodb GUI using API token?

Thanks and Regards.
Sumathy Venugopal

**Comment by wingkwong** [comment]
What is the use case for that?

**Reply by MGW-SumathyV** [reply]
I wanted to open the Nocodb GUI (project list page) from my application without asking user to signin.

**Reply by wingkwong** [reply]
You can fetch Project list data from Project API using API token and render in your application.

**Reply by MGW-SumathyV** [reply]
Should I be developing my UI and Nocodb GUI cannot be used in that case right? could you please confirm
